pkg/kubelet/container/runtime.go

```diff
@@ -364,14 +364,23 @@ type PortMapping struct {
 	HostIP string
 }
 
+type DeviceInfo struct {
+	// Path on host for mapping
+	PathOnHost string
+	// Path in Container to map
+	PathInContainer string
+	// Cgroup permissions
+	Permissions string
```

Comment:
I didn't find we define cgroup devices elsewhere. And regarding the struct I defined, it is same as ```github.com/docker/engine-api/types/container.DeviceMapping```, it's used in docker_manager.go
```go
Resources: dockercontainer.Resources{
			Memory:     memoryLimit,
			MemorySwap: -1,
			CPUShares:  cpuShares,
			Devices:    devices,
		},
```
But I want to put it here for both Docker and RKT, so they both could support more devices in the future. Also the current devices type ```[]string``` is not suit for Docker. It always need a host/container path mapping.